packages/core/src/app.ts

```diff
@@ -1520,6 +1520,8 @@ export class App {
             attachments?: AttachmentInfo[];
             auditResults?: AuditResult[];
             lastAudited?: Date;
+            expiresBy?: Date;
+            expiryFrequency?: ExpiryFrequencyOption;
```

Comment:
Better use a generic interval, like number of days. In the UI there can still be a list of predefined intervals but in the data layer there is no reason to restrict ourselves.

Reply:
That's a good idea. I'll implement that tomorrow.